source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSourceCompactStyle.mm

```diff
@@ -60,7 +60,13 @@ - (instancetype)initWithInputChoiceSet:(std::shared_ptr<AdaptiveCards::ChoiceSet
             }
             _defaultString = [titleArray componentsJoinedByString:@", "];
         } else {
-            _defaultString = @"";
+            NSString *firstChoice = nil;
+            if(!_choiceSetInput->GetChoices().empty()){
+                firstChoice = [NSString stringWithCString:(_choiceSetInput->GetChoices().begin()[0])->GetTitle().c_str() encoding:NSUTF8StringEncoding];
```

Comment:
>.begin() [](start = 88, length = 8)

You can remove this call to begin().

Reply:
Thanks

---
In reply to: [202846784](https://github.com/Microsoft/AdaptiveCards/pull/1709#discussion_r202846784) [](ancestors = 202846784)